### DISCUSSION TITLE
3.1.0-canary.6

### DISCUSSION THREAD

**Discussion by LabhanshAgrawal** [vercel/hyper#5685] [Announcements]
Fixed Term focus getting lost on clicking the header or dragging the window (#5678)

|   OS    | Installer |                                                                                                                   |                                                                                                                    |
| :-----: | :-------: | :---------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------: |
|   Mac   |    dmg    | [x64](https://github.com/vercel/hyper/releases/download/v3.1.0-canary.6/Hyper-3.1.0-canary.6-mac-x64.dmg) (Intel) | [arm64](https://github.com/vercel/hyper/releases/download/v3.1.0-canary.6/Hyper-3.1.0-canary.6-mac-arm64.dmg) (M1) |
|  Linux  |    deb    |      [x64](https://github.com/vercel/hyper/releases/download/v3.1.0-canary.6/hyper_3.1.0-canary.6_amd64.deb)      |     [arm64](https://github.com/vercel/hyper/releases/download/v3.1.0-canary.6/hyper_3.1.0-canary.6_arm64.deb)      |
|         |    rpm    |     [x64](https://github.com/vercel/hyper/releases/download/v3.1.0-canary.6/hyper-3.1.0-canary.6.x86_64.rpm)      |    [arm64](https://github.com/vercel/hyper/releases/download/v3.1.0-canary.6/hyper-3.1.0-canary.6.aarch64.rpm)     |
|         | AppImage  |      [x64](https://github.com/vercel/hyper/releases/download/v3.1.0-canary.6/Hyper-3.1.0-canary.6.AppImage)       |   [arm64](https://github.com/vercel/hyper/releases/download/v3.1.0-canary.6/Hyper-3.1.0-canary.6-arm64.AppImage)   |
|         |   snap    |     [x64](https://github.com/vercel/hyper/releases/download/v3.1.0-canary.6/hyper_3.1.0-canary.6_amd64.snap)      |                                                                                                                    |
| Windows |    exe    |      [x64](https://github.com/vercel/hyper/releases/download/v3.1.0-canary.6/Hyper-Setup-3.1.0-canary.6.exe)      |                                                                                                                    |

Note - Set `updateChannel` to `'canary'` in hyper config (`command/ctrl`+`,`) to stay on canary builds.

<hr /><em>This discussion was created from the release <a href='https://github.com/vercel/hyper/releases/tag/v3.1.0-canary.6'>3.1.0-canary.6</a>.</em>

**Comment by ivyreese** [comment]
Here's a weird one — I'm on an M1, and the arm64 canary.5 build was auto-updated to the x64 canary.6 build.

Just downloaded the arm64 build of 5 again, let it re-run the update, and confirmed that I was upgraded to the x64 build.

Before:
<img width="358" alt="canary 5" src="https://user-images.githubusercontent.com/2061924/122972492-72fc1e00-d34d-11eb-9fb0-414e228c23d6.png">

After:
<img width="337" alt="canary 6" src="https://user-images.githubusercontent.com/2061924/122972539-7d1e1c80-d34d-11eb-8068-db7bdbeee9d7.png">

Want me to file an issue?

**Reply by LabhanshAgrawal** [reply]
Have opened #5686

**Comment by ShikherVerma** [comment]
I cannot run DIR in hyper, I used to be able to do that in Command Prompt

**Reply by LabhanshAgrawal** [reply]
Which shell are you using with hyper?

**Reply by ShikherVerma** [reply]
The sheel is hyper that is why only I comment here

**Reply by LabhanshAgrawal** [reply]
Can you reply with your config? you can get it by doing `ctrl`+`,`

**Reply by ShikherVerma** [reply]
When I press Cantrol plus coma
![image](https://user-images.githubusercontent.com/10194699/123061932-b3c96500-d429-11eb-978f-e821e192748c.png)
why doesn't hyper implement a text editor, vscode is open source, you can include that inside into hyper

**Reply by duytrung** [reply]
Thanks  for the update, I wish this program has more frequent updates.

**Comment by GerardoArjona** [comment]
Hey guys, i've installed canary but I keep getting the message when opening: "Mac cannot open Hyper 2 because it cannot be checked for malicious software, software might need an update please contact developer"

**Reply by LabhanshAgrawal** [reply]
right click on it and click `open`

**Reply by GerardoArjona** [reply]
![image](https://user-images.githubusercontent.com/21064604/124534776-8b9f1600-ddda-11eb-924e-fbc81e93a322.png)

same thing

**Reply by LabhanshAgrawal** [reply]
can you post the link you downloaded it from
